.golangci.yml

```diff
@@ -0,0 +1,26 @@
+linters:
+  auto-fix: false
+  enable:
+    - nakedret 
+    - interfacer
+    - varcheck
+    - deadcode 
+    - structcheck 
+    - misspell
+    - maligned 
+    - ineffassign
+    - goconst
+    - goimports
+    - errcheck 
+    - dupl 
+    - unparam 
+    - golint 
+    - staticcheck
+    - unused
+    - gosimple
+    # missing to be added
+    # - lll (250), gocyclo, gosec, scopelint, unconvert, stylecheck
```

Comment:
```suggestion
```

Reply:
I do not think that we should add this comment. We do not know if we would like or not to ad them in the future.